sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -87,8 +87,11 @@ static RT::PiProgram createSpirvProgram(const RT::PiContext Context,
 }
 
 RT::PiProgram ProgramManager::getBuiltOpenCLProgram(OSModuleHandle M,
-                                                    const context &Context) {
-  RT::PiProgram &Program = m_CachedSpirvPrograms[std::make_pair(Context, M)];
+                                                 const context &Context) {
```

Comment:
Indentation needs fixed I think

Reply:
@alexbatashev, could you resolve this, please?

Reply:
Done